projects/canopy/src/lib/side-nav/side-nav-content/side-nav-content.component.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../../styles/mixins';
+
+.lg-side-nav-content {
+  margin: 0 0 var(--space-sm) 0;
+  width: 100%;
+
+  &:focus {
+    @include lg-inner-focus-outline(var(--default-focus-color));
+  }
+
+  @include lg-breakpoint('md') {
```

Comment:
Doh, sorry, another one that is not needed. Sorry, should try and spot these at the same time 🤦

Reply:
It happens 😄 I can re-thumb.

Reply:
Fixed.